src/model.ts

```diff
@@ -387,6 +389,7 @@ export class Model extends EventBus<any> implements CommandDispatcher {
       ...config,
       mode: config.mode || "normal",
       custom: config.custom || {},
+      defaultCurrencyFormat: config.defaultCurrencyFormat || "[$$]#,##0.00",
```

Comment:
We could also check if the given format is valid.

Reply:
indeed yes